backend/src/main/scala/cromwell/backend/validation/MemoryValidation.scala

```diff
@@ -83,7 +83,7 @@ class MemoryValidation(attributeName: String = RuntimeAttributesKeys.MemoryKey)
 
   override def key = attributeName
 
-  override def coercion = Seq(WomIntegerType, WomStringType)
+  override def coercion = Seq(WomIntegerType, WomStringType, WomLongType)
```

Comment:
Is the order important here? If so, I'd put Integer first, then Long, then String